src/ecalc/libraries/libecalc/common/libecalc/examples/simple/model_duplicate_names.yaml

```diff
@@ -0,0 +1,111 @@
+TIME_SERIES:
+  - NAME: SIM
+    FILE: production_data.csv
+    TYPE: DEFAULT
+FACILITY_INPUTS:
+  - NAME: genset
+    FILE: genset.csv
+    TYPE: ELECTRICITY2FUEL
+  - NAME: compressor_sampled
+    FILE: compressor_sampled.csv
+    TYPE: COMPRESSOR_TABULAR
+  - NAME: compressor_with_turbine_sampled
+    FILE: compressor_sampled_with_turbine.csv
+    TYPE: COMPRESSOR_TABULAR
+  - NAME: pump_sampled
+    FILE: pump_sampled.csv #!include, commented !include is now supported
+    TYPE: TABULAR
+  - NAME: pump_chart
+    FILE: pump_chart.csv
+    TYPE: PUMP_CHART_SINGLE_SPEED
+    UNITS:
+      HEAD: M
+      RATE: AM3_PER_HOUR
+      EFFICIENCY: PERCENTAGE
+
+FUEL_TYPES:
```

Comment:
is it only fuel_gas that is tested here? can the yaml test be made smaller to not include a lot other data that is not needed for the test?

Reply:
Yes only fuel_gas is tested. Have simplified the yaml-file now.

Also added verification for duplicate emissions within one fuel type, for the yaml.